### PR TITLE
fix: handle facilities with simultaneous "open" and "close" statuses

### PR DESCRIPTION
**Pull request by m0nhawk**
### Bug Fixes
* Fixes IDPH Facility ETL: some of the facilities simultaneously have both "open" and "closed" statuses. This fixes it, this PR also requires extra `status` field in `summary_location`.

### CODE REVIEW

File: covid19-etl/etl/idph_facility.py [hunk 1]
```diff
@@ -117,14 +119,15 @@ def parse_facility(self, date, facility):
...
-        status = facility["status"]
+        status = facility.get("status", "Open")
 
         summary_location_submitter_id = format_submitter_id(
             "summary_location",
             {
                 "country": self.country,
                 "state": self.state,
                 "facility_name": facility_name,
+                "status": status,
```
Comment (paulineribeyre) [inline covid19-etl/etl/idph_facility.py:130]: should this be reporting_org_status as well?
```suggestion
                "reporting_org_status": status,
```

File: covid19-etl/etl/idph_facility.py [hunk 2]
```diff
@@ -117,14 +119,15 @@ def parse_facility(self, date, facility):
         facility_name = facility["FacilityName"]
         confirmed_cases = facility["confirmed_cases"]
         deaths = facility["deaths"]
-        status = facility["status"]
+        status = facility.get("status", "Open")
```
Comment (paulineribeyre) [inline covid19-etl/etl/idph_facility.py:122]: why is the default "Open"? shouldn't we use None if we don't have the data?

Comment (m0nhawk) [reply covid19-etl/etl/idph_facility.py]: This is `None` now, I have checked with Sean.

File: covid19-etl/etl/idph_facility.py (hunk 1 above)
Comment (m0nhawk) [reply covid19-etl/etl/idph_facility.py]: Actually, `format_submitter_id` doesn't care about the keys in dictionary, it's for nice way to specify the values for `submitter_id`. But it's nice to have it coherent. 👍

Comment (paulineribeyre) [reply covid19-etl/etl/idph_facility.py]: oh right, woops

**Pull request merged**